Autopilot/Simulation/Intercepts/CommsBetweenStateMachines_Intercept.cpp

```diff
@@ -0,0 +1,67 @@
+#include "CommWithPathManager.hpp"
+#include "CommWithAttitudeManager.hpp"
+
+#include <fstream>
+
+/***********************************************************************************************************************
+ * Variables
+ **********************************************************************************************************************/
+
+static CommandsForAM commandsForAM = {};
+static AttitudeData attitudeData = {};
+
+static std::fstream PathToAttComLog;
+
+/***********************************************************************************************************************
+ * Prototypes
+ **********************************************************************************************************************/
+
+static void writeAttitudeCommandsToFile(CommandsForAM *commands);
+
+/***********************************************************************************************************************
+ * Code
+ **********************************************************************************************************************/
+
+void CommWithAMInit()
+{
+    PathToAttComLog.open("DebugLogs/PathToAttComLog.txt", std::fstream::out);
+    PathToAttComLog << "";
+    PathToAttComLog.close();
+
+}
+
+void SendCommandsForAM(CommandsForAM *commands)
+{
+    commandsForAM = *commands;
+
+    writeAttitudeCommandsToFile(commands);
+}
+
+bool GetAttitudeData(AttitudeData *data)
+{
+    *data = attitudeData;
+}
+
+void CommWithPMInit()
+{
+}
+
+void SendAttitudeData(AttitudeData *data)
+{
+    attitudeData = *data;
+}
+
+bool GetCommands(CommandsForAM *commands)
```

Comment:
`GetPMData()`?

Reply:
Yeah... I hate the naming too. but to change the name in the intercept, I have to change the name in the actual implementation file, which is already in devel. I didn't feel that renaming the files belonged in this PR. I'll create an asana entry for it though